src/main/java/bio/terra/common/stairway/StairwayProperties.java

```diff
@@ -14,44 +14,57 @@
   private boolean tracingEnabled;
   private Duration retentionCheckInterval;
   private Duration completedFlightRetention;
-  private String clusterNameSuffix;
-
-  public boolean isForceCleanStart() {
-    return forceCleanStart;
-  }
 
-  public boolean isMigrateUpgrade() {
-    return migrateUpgrade;
-  }
+  /**
+   * clusterNameSuffix is used to generate names when creating pubsub queues. It is not needed if
+   * the topicId and subscriptionId are provided.
+   */
+  private String clusterNameSuffix;
 
-  public int getMaxParallelFlights() {
-    return maxParallelFlights;
-  }
+  /** PubSub topic to use for stairway work queue. It must exist in the current GCP project */
+  private String gcpPubSubTopicId;
 
-  public Duration getQuietDownTimeout() {
-    return quietDownTimeout;
-  }
+  /**
+   * PubSub subscription to use for stairway work queue. It must exist in the current GCP project
```

Comment:
can we document that if these are null, we use the clusterNameSuffix to create topics/subscriptions?
Consider also renaming clusterNameSuffix to make it clear that it constrols creating a pubsub queue.

Reply:
I tried to improve the documentation in the properties file.